tests/samples/ReservedWords.scala

```diff
@@ -0,0 +1,51 @@
+import scala.scalanative._
+import scala.scalanative.native._
+import scala.scalanative.native.Nat._
+
+@native.link("ReservedWords")
+@native.extern
+object ReservedWords {
+  type `match` = native.CInt
+  type `object` = struct_object
+  type `type` = struct_object
+  type `lazy` = union_lazy
+  type `def` = `match`
+  type `finally` = struct_finally
+  type struct_object = native.CStruct2[`match`, native.CInt]
+  type struct_finally = native.CStruct2[`def`, `lazy`]
+  type union_lazy = native.CArray[Byte, Digit[_1, Digit[_2, _8]]]
+  def `with`(`sealed`: `match`, `implicit`: native.CInt, `forSome`: `lazy`): `type` = native.extern
+  def `implicit`(`type`: native.Ptr[`finally`]): `match` = native.extern
+}
+
+import ReservedWords._
+
+object ReservedWordsHelpers {
+
+  implicit class struct_object_ops(val p: native.Ptr[struct_object]) extends AnyVal {
+    def `yield`: `match` = !p._1
+    def `yield_=`(value: `match`):Unit = !p._1 = value
+    def `val`: native.CInt = !p._2
+    def `val_=`(value: native.CInt):Unit = !p._2 = value
+  }
+
+  def struct_object()(implicit z: native.Zone): native.Ptr[struct_object] = native.alloc[struct_object]
+
+  implicit class struct_finally_ops(val p: native.Ptr[struct_finally]) extends AnyVal {
+    def `val`: `def` = !p._1
+    def `val_=`(value: `def`):Unit = !p._1 = value
```

Comment:
These setters look a bit strange to me, maybe it is better to rename them to `set_<name>`?

Reply:
AFAIK, we cannot rename the methods, the `_=` suffix tells the Scala compiler that the method is a setter but the name needs to contain the variable name.